example/rds.tf

```diff
@@ -15,17 +15,22 @@ variable "cluster_state_bucket" {}
  *
  */
 module "example_team_rds" {
-  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.3"
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.5"
   cluster_name           = "${var.cluster_name}"
   cluster_state_bucket   = "${var.cluster_state_bucket}"
   team_name              = "example-repo"
   business-unit          = "example-bu"
   application            = "exampleapp"
   is-production          = "false"
+  db_engine_version      = "9.6"                                                                        # change this postgres version as you see fit.
```

Comment:
Since defaults are at version 10, should our examples reflect that as well?

Reply:
That would make sense, yes.